notebook/exploration-reports/2021.09-dag-pb-encryption.md

```diff
@@ -0,0 +1,202 @@
+# DAG-PB Encryption
```

Comment:
> Engage with folks who have more expertise in encryption

If you have ideas I'd be happy to invite them to the conversation.

> it would probably be good for us to get a bit more organised with some groups discussions/forums

Again, feel free to drag me into any relevant conversations. I'm just trying to move this forward.